client/webserver/site/src/js/wallets.ts

```diff
@@ -454,10 +454,20 @@ export default class WalletsPage extends BasePage {
     page.withdrawAvail.textContent = Doc.formatFullPrecision(wallet.balance.available, asset.info.unitinfo)
     page.withdrawLogo.src = Doc.logoPath(asset.symbol)
     page.withdrawName.textContent = asset.info.name
-    // page.withdrawFee.textContent = wallet.feerate
-    // page.withdrawUnit.textContent = wallet.units
     box.dataset.assetID = String(assetID)
     this.animation = this.showBox(box, page.walletPass)
+
+    Doc.hide(page.FeeRate)
+    const res = await postJSON('/api/getfeerate', {
+      assetID: assetID
+    })
+    if (!app().checkResponse(res) || res.feeRate === 0) {
+      page.withdrawFeeRate.textContent = 'unavailable'
+    } else if (res.feeRate > 0) {
+      page.withdrawFeeRate.textContent = res.feeRate
+      page.withdrawUnit.textContent = wallet.units + '/' + res.unit
```

Comment:
Would it be better if the fee rate was also just a constant connected to the wallet? It would not ever change, just like the units. I see dcr does right now, but can just divide by 1000 and use B.

Reply:
AFAICT, the fee rate changes and we would want to show the user a current fee rate?

Reply:
The rate changes but not the units. I mean the units are passed as a string, and don't need to be because we already know what they are per asset.

Reply:
> Would it be better if the fee rate

Oh, sorry. I just meant the unit. Not the rate...

Reply:
eth is gwei/gas dcr is atom/byte and btc is sat/byte and doesn't change.